src/account/AccountPickScreen.js

```diff
@@ -19,6 +20,14 @@ class AccountPickScreen extends React.Component {
     const { apiKey } = accounts[index];
     if (apiKey) {
       switchAccount(index); // Reset stream, message, user list
+
+      setTimeout(() => {
+        const { auth, fetchEvents, fetchUsersAndStatus, fetchMessages } = this.props;
+
+        fetchEvents(auth);
```

Comment:
These 3 lines are repeated in `MainScreen.js` so maybe we should refactor them out to avoid duplication